AdaptiveCards/sdk/rendering-cards/xamarin/android/adaptivecards-renderin-xamarin-android-renderer-actionhandler-icardactionhandler.md

```diff
@@ -103,15 +103,15 @@ public class MyCardActionHandler : ICardActionHandler
 --- 
 
 ### <a id="onmediastop"></a> OnMediaStop
-<p style='text-align:right'>Added in version 0.1</p>
+<p>Added in version 0.1</p>
 
 ```csharp
 void OnMediaStop (BaseCardElement p0, RenderedAdaptiveCard p1)
 ```
 
 Listener called when the media element stops playing.
 
-| Parameters | |
+| Parameters | &nbsp; |
```

Comment:
Could you please provide some context around why we need to use the &nbsp; representation for a space?

Reply:
@rahulamlekar On the nbsp, the column needs a header or it triggers a validation warning in the build report. If you prefer a column header, please let me know an appropriate value.